packages/arb-token-bridge-ui/src/state/app/utils.ts

```diff
@@ -84,6 +85,64 @@ export const transformDeposits = (
   })
 }
 
+export const transformDeposit = (tx: Deposit): MergedTransaction => {
```

Comment:
Created `transformDeposit` and `transformWithdrawal` that replaces `transformDeposits` and `transformWithdrawals`. 

We want to be able to map each tx individually instead of passing an array of txs.

I haven't deleted the old methods yet because there's a lot of dependencies and code that have to be removed, and didn't want to have extra diff there.

Reply:
I think extracting `transformDeposit` and `transformWithdrawal` as a separate PR (before this one) might make the diff smaller